### PR TITLE
Create secure in-memory directory on macOS

### LINKED ISSUE #29: `gopass edit` should create an in-memory temporary directory on macOS
MacOS doesn't have a `/dev/shm` directory so `gopass` falls back to `ioutil.TempDir` which will write plaintext to the disk. `pass` does a lot of work on [darwin] to avoid this.

[darwin]: https://git.zx2c4.com/password-store/tree/src/platform/darwin.sh#n18

### PR DESCRIPTION
**Pull request by dominikschulz**
Fixes #29

Note: Untested on MacOS so far. May need some tweaking.

### CODE REVIEW

**Comment by dominikschulz** [comment]
Tested on macOS. After a few small fixes it works fine for me.

File: fsutil/tempdir_darwin.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+package fsutil
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/cenkalti/backoff"
```
Comment (zwopir) [inline fsutil/tempdir_darwin.go]: we probably want to vendor github.com/cenkalti/backoff?

Comment (dominikschulz) [inline fsutil/tempdir_darwin.go]: Yes, definitely. Good catch.

**Comment by dominikschulz** [comment]
Signal handling is IMHO out of scope for this PR, but we should have a look at that later.

**Pull request merged**